src/Assets.php

```diff
@@ -140,6 +140,7 @@ public static function print_script_block_data() {
 			'hasTags'           => $tag_count > 0,
 			'productCategories' => $product_categories,
 			'homeUrl'           => esc_js( home_url( '/' ) ),
+			'productCount'      => array_sum( (array) $product_counts ),
```

Comment:
Shouldn't this only count the published products?
```suggestion
			'productCount'      => $product_counts->publish,
```

Reply:
The placeholder message infers that 'no' products have been created. `publish` does not cover private, future, and drafts -- posts the admin creating the block may be able to see.

Reply:
Makes sense. :+1: